Spinner/Spinner.module.less

```diff
@@ -0,0 +1,196 @@
+// Spinner.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/variables.less";
+@import "../styles/skin.less";
+
+@keyframes spinner {
+	0% {
+		opacity: 1;
+	}
+	100% {
+		opacity: 0;
+	}
+}
+
+.spinner {
+	.bg {
+		display: inline-block;
+		width: @agate-spinner-size;
+		height: @agate-spinner-size;
+		position: relative;
+
+		.fan1,
+		.fan2,
+		.fan3,
+		.fan4,
+		.fan5,
+		.fan6,
+		.fan7,
+		.fan8,
+		.fan9,
+		.fan10,
+		.fan11,
+		.fan12 {
+			transform-origin: @agate-spinner-transform-origin @agate-spinner-transform-origin;
+			will-change: opacity;
+			animation: spinner 1.2s linear infinite;
+
+			&:after {
+				content: " ";
+				display: block;
+				position: absolute;
+				top: 0;
+				left: 23px;
```

Comment:
Let's have these values (this one and the two following) be relative or calculated based on the size of the spinner for some more versatility.